misc/python/materialize/cli/mzbench.py

```diff
@@ -79,11 +81,12 @@ def main(args: argparse.Namespace) -> None:
     else:
         git_references = [None, *args.git_references]
 
-    if args.verbose:
-        build_tags = [None, *[mzbuild_tag(ref) for ref in args.git_references]]
-        print(f"DEBUG: num_iterations={args.num_measurements}")
-        print(f"DEBUG: worker_counts={worker_counts}")
-        print(f"DEBUG: mzbuild_tags={build_tags}")
+    ui.Verbosity.init_from_env(explicit=args.quiet)
+    dbg = ui.speaker("DEBUG: ")
+    build_tags = [None, *[mzbuild_tag(ref) for ref in args.git_references]]
+    dbg("num_iterators={nm}".format(nm=args.num_measurements))
+    dbg("worker_counts={wc}".format(wc=worker_counts))
+    dbg("mzbuild_tags={bt}".format(bt=build_tags))
```

Comment:
style nit: this is fine, but if there are no complex expressions (where complex means includes at least one non-dot operator) in the format variables we usually use f-strings:

```suggestion
    dbg(f"num_iterators={args.num_measurements}")
    dbg(f"worker_counts={worker_counts}")
    dbg(f"mzbuild_tags={build_tags}")
```

Reply:
Great, TIL about fstrings.